category/domain/category.go

```diff
@@ -128,6 +129,25 @@ func (a Attributes) All() []Attribute {
 	return att
 }
 
+// ToString returns a concatenated string of all the values of all attributes
+func (a Attributes) ToString() string {
```

Comment:
Do we really need a function that returns all Attributes Values as a list?
e.g. you have:

colour: red,green
size: xl,L

Ans then it would return a string "red,green,xl,l"
for what?

Reply:
Good point, team requested that, so I did. Just discussed through it, and there is no strong reason why, I remove it